workspace/welcome-to-xod/007-booleans/patch.xodp

```diff
@@ -1,15 +1,29 @@
 {
   "comments": [
     {
-      "content": "# Booleans\n\nIn previous lessons we mostly dealt with numbers (like counter increment and clock interval). Now let's talk about *booleans*. They have only 2 possible values — *true* or *false*, but are extremely useful and common. Indicating whether some condition is met, should a device be on or off, did some operation succeed — that's a job for a *boolean*!\n\n## Instructions\n\n1. Read the descriptions for `and`, `or` and `not` nodes.\n2. Guess what the output value of `and` will be.\n3. Run the simulation to confirm it.\n4. Use `tweak-boolean` nodes to change it to the opposite value.\n5. Check out descriptions for `nand`, `nor` and `xor` nodes.",
+      "content": "# Booleans\n\nIn previous lessons we mostly dealt with numbers adjusting counter step and clock interval. Now let's talk about *booleans*. They have only who possible values:\n\n- *True* (aka logic one, high signal)\n- *False* (aka logic zero, low signal)\n\nBooleans are extremely useful and common. They may be used to indicate whether some condition is met, should a device be on or off, did some operation succeed — that's a job for a boolean!\n\n## Exercise\n\nLet’s learn how boolean values can be combined.\n\n1. Read the descriptions for `and`, `or` and `not` nodes.\n2. Guess what the output value of `and` will be.\n3. Run the simulation to confirm it.\n4. Use `tweak-boolean` nodes to change it to the opposite value.\n\nCheck out descriptions for `nand`, `nor` and `xor` nodes. Add them onto the patch along with some additional tweaks to make a more complex processing.",
```

Comment:
```suggestion
      "content": "# Booleans\n\nIn previous lessons we mostly dealt with numbers adjusting counter step and clock interval. Now let's talk about *booleans*. They have only who possible values:\n\n- *True* (aka logic one, high signal)\n- *False* (aka logic zero, low signal)\n\nBooleans are extremely useful and common. They may be used to indicate whether some condition is met, should a device be on or off, did some operation succeed, etc.\n\n## Exercise\n\nLet’s learn how boolean values can be combined.\n\n1. Read the descriptions for `and`, `or` and `not` nodes.\n2. Guess what the output value of `and` will be.\n3. Run the simulation to confirm it.\n4. Use `tweak-boolean` nodes to change it to the opposite value.\n\nCheck out descriptions for `nand`, `nor` and `xor` nodes. Add them onto the patch along with some additional tweaks to make a more complex processing.",
```

I think "that's a job for a boolean"  remark became obsolete after rewording the rest.